src/main/scala/com/cognite/spark/datasource/DefaultSource.scala

```diff
@@ -70,6 +77,7 @@ class DefaultSource
       case None => ""
     }
     val collectMetrics = toBoolean(parameters, "collectMetrics")
+    val saveMode = parameters.getOrElse("savemode", "write")
```

Comment:
I don't see "write" in the `config.saveMode match` below, shouldn't this be set to one of the values in that `match`?

Reply:
Write was just a working name I forgot to change :)